ads/_config.js

```diff
@@ -267,6 +267,10 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  directadvert: {
+    renderStartImplemented: true,
```

Comment:
I see you have renderStart implemented, but when I test locally there's no `render-start` message received. Could you please double check? Thanks

Reply:
I removed this option for directadvert

Reply:
Note that we do recommend implementing renderStart API as it provides better UX. Please consider implementing it. 
Please refer to our [doc](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis) and let me know if there's any question.